pilot/pkg/kube/inject/inject.go

```diff
@@ -505,6 +505,26 @@ func isset(m map[string]string, key string) bool {
 	return ok
 }
 
+func defaultAsFalse(m map[string]string, key string) bool {
```

Comment:
Not sure we need this - see the other methods. 

You can use {{ annotation .... values.global.splitInbound }}

Reply:
Let me rephrase:  values.yaml should only have the "includeInboundPorts: *"  - we can bikeshed on the name or if it's bool or not. 

But no iptables internal port in the setting - we don't set the 15001 and we don't guarantee we'll use it.

Reply:
And my point is that we have 'annotation or value' - please use that, don't change inject.go.

Reply:
Fixed: Using the existing includeInboundPorts="*" to enable the feature